cdash/CTestConfig.cmake

```diff
@@ -0,0 +1,8 @@
+set(CTEST_PROJECT_NAME "Charm%2B%2B")
+set(CTEST_NIGHTLY_START_TIME "01:00:00 UTC")
```

Comment:
1 am UTC would be 7 or 8 pm local time.

Reply:
I think this parameter doesn't really influence anything. In any case, time zones vary for each testing location.